tests/WebChromiumTest.php

```diff
@@ -21,6 +21,7 @@ class WebChromiumTest extends Base
         ...Base::BAR_RESPONSES,
         ...Base::GENERAL_RESPONSES,
         ...Base::LARGE_FILE_RESPONSES,
+        "https://appwrite.io/v1/mock/tests/general/download?project=console",
```

Comment:
Why?

Reply:
@lohanidamodar The Web SDK returns a URL instead of the contents of the URL